streaming/base/format/mds/encodings.py

```diff
@@ -464,7 +464,29 @@ def decode(self, data: bytes) -> Image.Image:
 
 
 class JPEG(Encoding):
-    """Store PIL image as JPEG."""
+    """Store PIL image as JPEG. Optionally specify quality."""
+
+    def __init__(self, quality: int = 75):
+        if not isinstance(quality, int):
+            raise ValueError('JPEG quality must be an integer')
+        if not (0 <= quality <= 100):
+            raise ValueError('JPEG quality must be between 0 and 100')
+        self.quality = quality
+
+    @classmethod
+    def from_str(cls, config: str) -> Self:
```

Comment:
We should probably also add a test for this too then, to also confirm what it should look like when used. Thanks!